src/app/routes/assets/asset-enrolment-list/asset-enrolment-list.component.spec.ts

```diff
@@ -1,25 +1,114 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 
 import { AssetEnrolmentListComponent } from './asset-enrolment-list.component';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ActivatedRoute, Params } from '@angular/router';
+import { BehaviorSubject } from 'rxjs';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+import { UrlService } from '../../../shared/services/url-service/url.service';
+import { AssetDetailsActions, AssetDetailsSelectors } from '@state';
+import { RouterTestingModule } from '@angular/router/testing';
 
-xdescribe('AssetEnrolmentListComponent', () => {
+export class MockActivatedRoute {
```

Comment:
Is this class a generic class that could be shared by multiple tests?

Reply:
Yeah, I'm planning to do a directory with mocks/stubs/function supports for tests. This require some bigger refactoring in multiple places.